frontend/src/components/Post.jsx

```diff
@@ -45,7 +45,7 @@ export const Post = (props) => {
                     }
                 />
             </ListItem>
-            <Like id={props.post.id} />
+            <Like id={props.post.liked} />
```

Comment:
Need to pass in 2 separate props, one is id={props.post.id} and other is liked={props.post.liked}